### PR TITLE
Eoran Bud Flaw (and improvements)

### PR DESCRIPTION
**Pull request by OctusGit**
## About The Pull Request

A characther flaw that starts you with an Eoran bud on your head, removing it adds stress.
Also makes it unable to remove the Eoran bud from other people with the plant if you also wear it.

## Why It's Good For The Game

Its pretty much forced pacifism or consequences. Being a pacifist in itself is already a big vulnerability in Stonekeep. As for the inability to remove others plants if you wear one, its just to stop an oversight with two controlled pacifist just casually removing each others plant.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by sanshoom** [comment]
So called 'pacifists' after I remove the bud from their head during their sleep and burn it.

File: code/datums/character_flaw/_character_flaw.dm [hunk 1]
```diff
@@ -269,3 +270,32 @@ GLOBAL_LIST_INIT(character_flaws, list("Alcoholic"=/datum/charflaw/addiction/alc
 		if(H.name) // If you don't check this, the log entry wont have a name as flaw_on_life is checked at least once before the name is set.
 			log_hunted("[H.ckey] playing as [H.name] had the hunted flaw by vice.")
 			logged = TRUE
+
+/datum/charflaw/budaddict
+	name = "Bud Addict"
+	desc = "Im too used to wearing an eoran bud, I couldnt live without it."
```
Comment (BadAtThisGame302) [inline code/datums/character_flaw/_character_flaw.dm:276]: I'm so used.... I can't live...

File: code/datums/stress/negative_events.dm [hunk 2]
```diff
@@ -293,3 +293,8 @@
 	timer = 3 MINUTES
 	stressadd = 6
 	desc = "<span class='red'>I slept on the floor! What am I?! An animal?!</span>"
+
+/datum/stressevent/nobud
+	timer = 1 MINUTES
+	stressadd = 20
+	desc = "<span class='red'>I cant think straight! I need my eoran bud again, where is it?!</span>"
```
Comment (BadAtThisGame302) [inline code/datums/stress/negative_events.dm:300]: I can't*

File: code/datums/character_flaw/_character_flaw.dm (hunk 1 above)
Comment (OctusGit) [reply code/datums/character_flaw/_character_flaw.dm:276]: i failed english class

File: code/datums/stress/negative_events.dm (hunk 2 above)
Comment (OctusGit) [reply code/datums/stress/negative_events.dm:300]: its bleak

File: code/datums/character_flaw/_character_flaw.dm (hunk 1 above)
Comment (sanshoom) [reply code/datums/character_flaw/_character_flaw.dm:276]: > i failed english class

we know

**Comment by AceVX** [comment]
I'm not sure about this one balance wise. since flaws are meant to be like an addiction or habit, something you have to maintain ever so often.

An eoran bud would just be a one and done acquisition for the entire round. Doesn't really have any drawbacks other than what the bud does on its own.

**Comment by OctusGit** [comment]
> I'm not sure about this one balance wise. since flaws are meant to be like an addiction or habit, something you have to maintain ever so often.
> 
> An eoran bud would just be a one and done acquisition for the entire round. Doesn't really have any drawbacks other than what the bud does on its own.

The drawbacks are pacifism or you get bad mood debuffs. Also lost limbs and cyclops exist.

**Comment by BadAtThisGame302** [comment]
> I'm not sure about this one balance wise. since flaws are meant to be like an addiction or habit, something you have to maintain ever so often.
> 
> An eoran bud would just be a one and done acquisition for the entire round. Doesn't really have any drawbacks other than what the bud does on its own.

Kinda like needing spectacles, you even start with them.

**Comment by AceVX** [comment]
> > I'm not sure about this one balance wise. since flaws are meant to be like an addiction or habit, something you have to maintain ever so often.
> 
> > 
> 
> > An eoran bud would just be a one and done acquisition for the entire round. Doesn't really have any drawbacks other than what the bud does on its own.
> 
> 
> 
> Kinda like needing spectacles, you even start with them.

Valid fs, I take it back

**Comment by NPC1314** [comment]
This is pretty sound (it seems functional unlike say, arsonist flaw, and kinda balanced)

Theres this thing tho.... random flaw, new players getting this is a big yikes

**Comment by NPC1314** [comment]
Stale, marked for death

**Pull request closed without merging**